frontend/sideMenu/sideMenu.component.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+(function() {
+    angular
+    .module('app')
+    .component('sideMenu', {
+        templateUrl: "app/sideMenu/sideMenu.html",
+        controller: sideMenuController,
+        controllerAs: "sideMenuCtrl",
+        bindings: {
+            type: '@'
+        }
+    });
+
+    function sideMenuController(AuthService, STATES, $state, $mdSidenav, 
+        $mdDialog, HomeItemsFactory, ManageInstItemsFactory, InstitutionService) {
+        
+        const sideMenuCtrl = this;
+        const HOME_TYPE = "HOME";
+        const MANAG_INST_TYPE = "MANAGE_INSTITUTION";
+        const INST_PAGE_TYPE = "INSTITUTION_PAGE";
+
+        sideMenuCtrl.user = AuthService.getCurrentUser();
+        
+        sideMenuCtrl.$postLink = () => {
+            setup();
+        };
+        
+        const setup = () => {
+            if(sideMenuCtrl.isType(HOME_TYPE)) {
+                sideMenuCtrl.entity = sideMenuCtrl.user;
+                loadItems();
+            } else {
+                const instKey = $state.params.institutionKey;
+                InstitutionService.getInstitution(instKey)
+                .then(inst => {
+                    sideMenuCtrl.entity = new Institution(inst);
+                    loadItems();
+                }).catch(err => console.error(err));
+            }
+        };
+
+        const loadItems = () => {
```

Comment:
you can create an object instead of constant that has the type and loadItems function.
So you can just call sideMenuCtrl.type.loadItems.
Maybe choose another "name" for the variable type, to make reading better.